static/js/message_edit_history.js

```diff
@@ -22,7 +22,6 @@ export function fetch_and_render_message_history(message) {
         success(data) {
             const content_edit_history = [];
             let prev_time = null;
-
```

Comment:
Please remove this whitespace change; I'm pretty sure this blank line was there for readability.